src/layers/core/arc-layer/arc-layer.js

```diff
@@ -32,13 +32,12 @@ const defaultProps = {
   getTargetPosition: x => x.targetPosition,
   getSourceColor: x => x.color || DEFAULT_COLOR,
   getTargetColor: x => x.color || DEFAULT_COLOR,
-  strokeWidth: 1,
-  fp64: false
+  strokeWidth: 1
 };
 
 export default class ArcLayer extends Layer {
   getShaders() {
-    return this.props.fp64 ? {
+    return this.props.fp64 && this.props.projectionMode === 1 ? {
```

Comment:
Currently when using 64-bit under meter offset mode, the 64-bit prop will be ignored and the layer will be rendered in 32-bit. I agree that we should output an warning message in the console, but throwing an exception might not be what users want.